src/components/organisms/MessageBox/MessageBox.tsx

```diff
@@ -27,6 +27,20 @@ const MessageBox: React.FC = () => {
     [alert]
   );
 
+  const modify = (title: string) => {
+    const wordsArray = title.split(' ');
+    const unauthIndex = wordsArray.indexOf('UNAUTHORIZED');
+
+    if (unauthIndex === -1) {
+      return title;
+    }
+
+    return `We're sorry, it looks like you're not authorized to connect to this cluster. Please take a look at our troubleshooting guide for cluster connections in our documentation https://kubeshop.github.io/monokle/cluster-issues for steps on how to resolve this issue.\n\n
```

Comment:
To make the link work properly with Markdown, we should use the markdown link notation `[text](link)`
```suggestion
    return `We're sorry, it looks like you're not authorized to connect to this cluster. Please take a look at our [troubleshooting guide for cluster connections in our documentation](https://kubeshop.github.io/monokle/cluster-issues/) for steps on how to resolve this issue.\n\n
```